store/sql_store.go

```diff
@@ -113,7 +88,7 @@ func (ss *SQLStore) Spinmint() SpinmintStore {
 }
 
 func (ss *SQLStore) DropAllTables() {
-	err := ss.master.TruncateTables()
+	err := multiExec(ss.db, []string{"TRUNCATE TABLE Issues", "TRUNCATE TABLE PullRequests", "TRUNCATE TABLE Spinmint"})
```

Comment:
I don't see much benefit in refactoring this. It can be inside this method itself.

Reply:
Updated